src/parse.rs

```diff
@@ -331,6 +331,17 @@ fn handle_unilateral<'a>(
         Response::Expunge(n) => {
             unsolicited.send(UnsolicitedResponse::Expunge(n)).unwrap();
         }
+        Response::Data {
+            status,
+            code,
+            information,
+        } if imap_proto::Status::Ok == status && code.is_none() => {
```

Comment:
Is it necessary for the code to `None` of this to be considered informational? From what I can tell, it's enough for the response to be untagged for it to be considered unsolicited — or am I missing something?

Reply:
In which case we should include the `code` in the unsolicited response type.